elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/ElasticSearchAdapterTest.java

```diff
@@ -705,6 +737,97 @@ public void approximateCount() throws Exception {
             "state=AL; EXPR$1=3; EXPR$2=3",
             "state=AR; EXPR$1=3; EXPR$2=3");
   }
+  /**
+   * Test the ElasticSearch match query. The match query is translated from CONTAINS query which
```

Comment:
I didn't expect this test to be so long. Can you pls move it into separate class (similarly to `AggregationTest`) ? There you can create any mapping you want

Reply:
Done. Moved it to new class MatchTest